driver/kubernetes/driver.go

```diff
@@ -165,16 +165,18 @@ func (d *Driver) Stop(ctx context.Context, force bool) error {
 	return nil
 }
 
-func (d *Driver) Rm(ctx context.Context, force bool, rmVolume bool) error {
-	if err := d.deploymentClient.Delete(ctx, d.deployment.Name, metav1.DeleteOptions{}); err != nil {
-		if !apierrors.IsNotFound(err) {
-			return errors.Wrapf(err, "error while calling deploymentClient.Delete for %q", d.deployment.Name)
-		}
-	}
-	for _, cfg := range d.configMaps {
-		if err := d.configMapClient.Delete(ctx, cfg.Name, metav1.DeleteOptions{}); err != nil {
+func (d *Driver) Rm(ctx context.Context, force bool, rmVolume bool, rmBuildkit bool) error {
+	if rmBuildkit {
```

Comment:
```suggestion
	if !rmBuildkit {
		return nil
	}
```

Reply:
If I understand correctly, this is doing nothing in this implementation when `rmBuildkit` is false. It's weird to call this method in that case 🤔